run.py

```diff
@@ -82,8 +82,25 @@ def current_request():
         else:
             logger.info("Schema exists.", schema=db_schema)
 
-            logger.info("Running Alembic database upgrade")
-            command.upgrade(alembic_cfg, "head")
+            # START 15-Feb-2019 Bug Fix 751
```

Comment:
Why do we need to put in a log line saying that the alembic scripts are not being run if they have been removed from the code?

Reply:
I ummed and arr'd about that one . Since it is a temporary fix I was minded to leave the comment there , which is not something I normally advocate. Will bring up with John in t morning